data/human/culture republic.txt

```diff
@@ -11,28 +11,118 @@
 # "Missions" which add culture events to the spaceport.
 
 #mission "cultural holiday template"
-	landing
+#	minor
+#	source
+#		government "placeholder"
+#		attributes "placeholder"
+#		not attributes "station"
+#	to offer
+#		and
+#			month == placeholder
+#			day >= placeholder
+#			day <= placeholder
+#	on offer
+#		conversation
+#			`placeholder`
+#				fail
+			
+#mission "cultural random template"
+#	minor
+#	source
+#		government "placeholder"
+#		attributes "placeholder"
+#		not attributes "station"
+#	to offer
+#		random == 1
+#	on offer
+#		conversation
+#			`placeholder`
+#				fail
+
+
+#############################################
+#           Republic or Syndicate           #
+#############################################
+
+mission "Sad Gamblers"
+	minor
+	source
+		or
+			and
+				government "republic"
+				attributes "near earth"
+			government "syndicate"
+		not attributes "station"
 	to offer
-		and
-			month == placeholder
-			day >= placeholder
-			day <= placeholder
+		random == 1
+	on offer
+		conversation
+			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in an 12th division win (a day or two worth of standard pay).`
+			`	It's difficult not to pity them as they immediately buy more tickets with the whole value.`
+				fail
+
+#############################################
+#               Republic Only               #
+#############################################
+
+mission "Gift Store Interaction"
+	minor
 	source
-		government "placeholder"
-		attributes "spaceport"
+		government "republic"
+		attributes "deep"
+		not attributes "station"
+	to offer
+		random == 1
 	on offer
-		dialog
-		` placeholder`
-			fail
-			
-#mission "cultural random template"
-	landing
+		conversation
+			`As you wander the spaceport with a snack, waiting for your ship to be refuelled, you pass a local souvenir shop and poke your head in to kill a little time. You see a decent looking mug with the text "<source>" printed underneath an attractive, stylised representation of the spaceport you are presently standing in and pick it up to see how it feels. The handle is solid, and not too small for your fingers, and the lip of it looks like it would be comfortable to drink out of.`
```

Comment:
refuelled => refueled

Reply:
Bloody american spellings. :P  Don't get too hung up on typos at this stage though, it hasn't been subjected to a proper spellcheck yet. (My initial ideas are on the wong computer.)